ruby/README.md

```diff
@@ -56,6 +56,41 @@ inherit_from: rails_rubocop.yml
 # project-specific configuration...
 ```
 
+## Reek
+
+Every project should start with the `reek.yml` present here. It should
```

Comment:
Guess we should keep the dot to avoid confusion.

```suggestion
Every project should start with the `.reek.yml` present here. It should
```

Reply:
I think it's referring to the file in this repo, though, not the name it'll get when pulled into a project.